jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -130,6 +139,11 @@
    */
   public static final String JAEGER_DISABLE_GLOBAL_TRACER = JAEGER_PREFIX + "DISABLE_GLOBAL_TRACER";
 
+  /**
+   * Boolean to determine if trace context should be propagated also using B3 format. Default is false.
+   */
+  public static final String JAEGER_B3_CODEC = JAEGER_PREFIX + "B3_CODEC";
```

Comment:
The description is a bit misleading, since it's not replacing the default Uber-Trace-Id format with B3, but _adds_ B3 support in addition to the default. If we are going to support Trace-Context in the future, should this setting perhaps be a list of formats instead of a true/false?

Reply:
It does say "also using B3 format", but can clarify the description. It is likely we will only have to support B3 and Trace-Context as the extra formats, so not sure a list vs two boolean properties is a big deal - but happy to change if you would prefer?

We may actually want to support Trace-Context along with Jaeger format always, rather than based on config - in which case only B3 would be configurable?

Reply:
The reason I prefer a list is because it's self explanatory as a configuration option. E.g. `JAEGER_B3=true` doesn't imply whether Uber-Trace-Id is included or not (without consulting documentation), but `JAEGER_PROPAGATION=uber,b3,w3c` or `JAEGER_PROPAGATION=b3` is pretty clear. And it also allows B3-only, which some people might want if they have a Zipkin-centric installation already.